packages/xo-server/src/xo-mixins/rest-api.mjs

```diff
@@ -1514,6 +1514,19 @@ export default class RestApi {
         res.sendStatus(200)
       })
     )
+
+    api.post(
+      '/:collection(users)',
+      json(),
+      wrap(async (req, res) => {
+        const { name, password, permission } = req.body
+        if (!name || !password) {
```

Comment:
Explicit condition to ensure that we are testing for `null/undefined` and not for a falsy value or a boolean.
```suggestion
        if (name != null || password != null) {
```